examples/text-classification/run_glue.py

```diff
@@ -353,20 +391,29 @@ def compute_metrics(p: EvalPrediction):
         train_result = trainer.train(
             model_path=model_args.model_name_or_path if os.path.isdir(model_args.model_name_or_path) else None
         )
-        metrics = train_result.metrics
```

Comment:
This line should not be removed.

Reply:
Thank you for your clarification, I ran into an issue but if this just hasn´t it be released, I revert it.